### PR TITLE
User can delete an entry - Bryan Bryan

### PR DESCRIPTION
**Pull request by bbrown949**
[Screen Recording 2023-04-23 at 5.40.32 PM.zip](https://github.com/bbrown949/code-journal/files/11305434/Screen.Recording.2023-04-23.at.5.40.32.PM.zip)

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -81,3 +81,80 @@ ul {
...
+
+.delete-button {
+  font-size: 14px;
+  padding-left: 11px;
+  margin: 0;
+  position: absolute;
+  bottom: 12px;
+  color: red;
+  padding-top: 15px;
+}
```
Comment (shawnkost) [inline css/styles.css]: This shouldn't be absolutely positioned. You should be utilizing flexbox

File: css/styles.css [hunk 2]
```diff
@@ -81,3 +81,80 @@ ul {
...
+  position: absolute;
+  bottom: 0;
+  top: 0;
+  right: 0;
+  left: 0;
+  background-color: rgba(33, 33, 33, 0.65);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (shawnkost) [inline css/styles.css]: This should be `position: fixed` instead & remove the `height` & `width` I believe

File: js/main.js [hunk 3]
```diff
@@ -172,8 +176,49 @@ $ul.addEventListener('click', function (event) {
...
-
       viewSwap('entry-form');
+
+      $deleteButton.className = 'delete-button';
     }
   }
 });
+
+// modal below
+const $deleteBtn = document.querySelector('.delete-button');
```
Comment (shawnkost) [inline js/main.js]: You already queried for this button before

File: js/main.js [hunk 4]
```diff
@@ -172,8 +176,49 @@ $ul.addEventListener('click', function (event) {
...
+const $popupModal = document.querySelector('.delete-entry-background');
+$deleteBtn.addEventListener('click', $deleteEntryPopup);
+function $deleteEntryPopup() {
```
Comment (shawnkost) [inline js/main.js]: `$` should be used alongside variables that hold DOM elements. They shouldn't be used for function names

File: js/main.js [hunk 5]
```diff
@@ -172,8 +176,49 @@ $ul.addEventListener('click', function (event) {
...
+  $popupModal.className = 'delete-entry-background';
+}
+
+// When the user clicks Cancel, hide the modal.
+const $cancelBtn = document.querySelector('.dlt-btn-cancel');
+$cancelBtn.addEventListener('click', $cancelDelete);
+function $cancelDelete() {
```
Comment (shawnkost) [inline js/main.js]: `$` should be used alongside variables that hold DOM elements. They shouldn't be used for function names

File: js/main.js [hunk 6]
```diff
@@ -172,8 +176,49 @@ $ul.addEventListener('click', function (event) {
...
+  $popupModal.className = 'delete-entry-background hide';
+}
+
+//
+
+const $confirmDeleteBtn = document.querySelector('.dlt-btn-confirm');
+$confirmDeleteBtn.addEventListener('click', $confirmDeletion);
+function $confirmDeletion() {
```
Comment (shawnkost) [inline js/main.js]: `$` should be used alongside variables that hold DOM elements. They shouldn't be used for function names

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good! A few changes requested below

File: css/styles.css (hunk 1 above)
Comment (bbrown949) [reply css/styles.css]: Yeah i did for the container of the two buttons. went ahead and removed this class here though. still looks good

File: css/styles.css (hunk 2 above)
Comment (bbrown949) [reply css/styles.css]: done

File: js/main.js (hunk 3 above)
Comment (bbrown949) [reply js/main.js]: done

File: js/main.js (hunk 4 above)
Comment (bbrown949) [reply js/main.js]: done

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**